OpenImis.ModulesV2/FeedbackModule/Models/FeedbackModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenImis.ModulesV2.FeedbackModule.Models
+{
+    public class FeedbackModel
```

Comment:
I suggest to change this to FeedbackResponseModel

Reply:
Done